udata/harvest/actions.py

```diff
@@ -162,3 +164,43 @@ def unschedule(ident):
     source.periodic_task.delete()
     signals.harvest_source_unscheduled.send(source)
     return source
+
+
+def attach(domain, filename):
+    '''Attach existing dataset to their harvest remote id before harvesting.
+
+    The expected csv file format is the following:
+
+    - a column with header "local" and the local IDs or slugs
+    - a column with header "remote" and the remote IDs
+
+    The delimiter should be ";". columns order
+    and extras columns does not matter
+    '''
+    count = 0
+    with open(filename) as csvfile:
+        reader = csv.DictReader(csvfile,
+                                delimiter=b';',
+                                quotechar=b'"')
+        for row in reader:
+            try:
+                dataset = Dataset.objects.get(id=ObjectId(row['local']))
+            except:  # noqa  (Never stop on failure)
```

Comment:
Might be interesting to have a counter for failures too? (I don't know if it happens often)